common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -179,12 +179,16 @@ div.problem {
 
   form > label, .problem-group-label {
     display: block;
-    margin-bottom: 0;
+    margin-bottom: $baseline;
     font: inherit;
     color: inherit;
     -webkit-font-smoothing: initial;
   }
 
+  .problem-group-label + .question-description {
+    margin-top: -20px;
```

Comment:
Seeing this hardcoded -20px scares me. Should it be based off of baseline? What is the margin-top value that it is overriding?